src/main/java/uk/gov/pay/api/resources/PaymentRefundsResource.java

```diff
@@ -93,6 +100,22 @@ public PaymentRefundsResource(Client client, PublicApiConfig configuration,
     @GET
     @Timed
     @Produces(APPLICATION_JSON)
+    @Operation(security = {@SecurityRequirement(name = "BearerAuth")},
+            operationId = "Get all refunds for a payment",
+            summary = "Return refunds for a payment. " +
+                    "The Authorisation token needs to be specified in the 'authorization' header as 'authorization: Bearer YOUR_API_KEY_HERE'",
+            responses = {
+                    @io.swagger.v3.oas.annotations.responses.ApiResponse(responseCode = "200", description = "OK",
+                            content = @Content(schema = @Schema(implementation = RefundForSearchResult.class))),
+                    @io.swagger.v3.oas.annotations.responses.ApiResponse(responseCode = "401", description = "Credentials are required to access this resource"),
+                    @io.swagger.v3.oas.annotations.responses.ApiResponse(responseCode = "404", description = "Not found",
+                            content = @Content(schema = @Schema(implementation = PaymentError.class))),
+                    @io.swagger.v3.oas.annotations.responses.ApiResponse(responseCode = "429", description = "Too many requests",
+                            content = @Content(schema = @Schema(implementation = ApiErrorResponse.class))),
+                    @io.swagger.v3.oas.annotations.responses.ApiResponse(responseCode = "500", description = "Downstream system error",
+                            content = @Content(schema = @Schema(implementation = PaymentError.class)))
+            }
```

Comment:
Fully qualified package name `io.swagger.v3.oas.annotations.responses.ApiResponse`  is required here as it conflicts with annotation we already use for current swagger plugin